wordpress-mobile/WordPress-iOS
# Fix retention cycles

Fixes #13097

Depends on https://github.com/wordpress-mobile/WordPress-iOS/pull/13085

### To test

1. Open Instruments
2. Run "Allocations"
3. Run the app
4. Select the app under Your Simulator Device > Running Applications: WordPress
5. Tap "Record" (the button with a red circle)
6. In the bottom sidebar, type "mediaed"
7. Open and close the Media Editor multiple times
8. In Instruments, check that the MediaEditor classes appear and then disappear when it's closed

Plus: test typing "tocrop" and checking that the crop view controller is correctly deallocated.

PR submission checklist:
